assets/src/stories-editor/components/block-mover/block-draggable.js

```diff
@@ -54,13 +90,79 @@ BlockDraggable.propTypes = {
 	children: PropTypes.any.isRequired,
 	onDragStart: PropTypes.func,
 	onDragEnd: PropTypes.func,
+	onNeighborHover: PropTypes.func,
+	onNeighborDrop: PropTypes.func,
+	getNeighborPageId: PropTypes.func,
```

Comment:
These prop types here can be marked `isRequired` as they should be guaranteed to exist

Reply:
Fixed in 04fffea.